ci

```diff
@@ -116,8 +167,12 @@ run_log_commit () {
   fi
   git push
   popd
-
+   
   if [[ $SLACK_FSTAR_WEBHOOK != "" ]]; then
+
+    # set var to false just to make sure not used wrong elsewhere
+    VALID_MEMBER=false
```

Comment:
I would generally advise against using global variables as the return value of a function. Some suggestions coming up on how to do this better.